contracts/carrot-app/src/replies/create_position.rs

```diff
@@ -12,12 +9,14 @@ use crate::{
 };
 
 pub fn create_position_reply(deps: DepsMut, env: Env, app: App, reply: Reply) -> AppResult {
+    let b = reply
+        .result
+        .unwrap()
+        .data
+        .expect("Failed to create position");
     // Parse the msg exec response from the reply
-    let authz_response: MsgExecResponse = reply.result.try_into()?;
-
-    // Parse the position response from the first authz message
-    let response: MsgCreatePositionResponse =
-        Binary(authz_response.results[0].clone()).try_into()?;
+    let parsed = cw_utils::parse_execute_response_data(&b)?;
+    let response: MsgCreatePositionResponse = from_json(parsed.data.unwrap_or_default())?;
```

Comment:
from_json?

Reply:
Can we set up a test that uses mainnet (instead of relying on the FE)

Reply:
> from_json?

Yes, it was wrong, thanks